src/components/DisplayResults.tsx

```diff
@@ -31,8 +31,8 @@ export const DisplayResults = ({
             <Text color="status-warning" size="xsmall">
               <MdError />{" "}
             </Text>
-            Hey, it looks like you've uploaded a picture that doesn't have a
-            plane in it!
+            Hey, it looks like you've uploaded a picture that doesn't have an
+            airplane in it!
```

Comment:
Aeroplane for British English?

Reply:
Fixed typo in the [1a3eed0](https://github.com/what-plane/what-plane-frontend/pull/28/commits/1a3eed0a6df6102dc85403b56d6340855623bb5c)